node/src/Commands.ts

```diff
@@ -804,6 +804,20 @@ const positiveInfinityArg = "+inf";
 const negativeInfinityArg = "-inf";
 const isInclusiveArg = "(";
 
+function getScoreLimitArg(score: ScoreLimit): string {
+    if (score == "positiveInfinity") {
+        return positiveInfinityArg;
+    } else if (score == "negativeInfinity") {
+        return negativeInfinityArg;
```

Comment:
If we have this function I don't see why we need to have the negativeInfinityArg , positiveInfinityArg and isInclusiveArg consts. We don't use them anywhere else, right?